packages/contracts/src/dollar/core/CreditNFT.sol

```diff
@@ -91,22 +92,24 @@ contract CreditNFT is ERC1155Ubiquity {
     function updateTotalDebt() public {
         bool reachedEndOfExpiredKeys = false;
         uint256 currentBlockNumber = _sortedBlockNumbers.popFront();
+        uint256 outstandingDebt = _totalOutstandingDebt;
 
         //if list is empty, currentBlockNumber will be 0
         while (!reachedEndOfExpiredKeys && currentBlockNumber != 0) {
             if (currentBlockNumber > block.number) {
                 //put the key back in since we popped, and end loop
-                _sortedBlockNumbers.pushFront(currentBlockNumber);
+                require(_sortedBlockNumbers.pushFront(currentBlockNumber));
```

Comment:
Also same as earlier `require` comment.

Reply:
Both the EnumerableSet and StructurededLinkList will return a bool for functions that alter the sets. Slither was calling these out as being ignored and a require was the simplest way to ensure a revert on failure.